docs/templates/registry.md

```diff
@@ -98,8 +103,36 @@ templateA/
     helper.py
 ```
 
-The template directories may be organized in any way that makes sense to the
-repository maintainers.
+In this example, `templateA` is a template directory, and `v1`, `v1.01`, and 
+`v1.1` are template version directories that hold the versions of `templateA`.
+
+### Registry based template references
+
+In general, 
+[templates references](https://github.com/kubernetes/deployment-manager/blob/master/docs/design/design.md#template-references)
+are just URLs to HTTP endpoints. However, because a template registry follows
+the conventions outlined above, references to templates in a template registry
+can be shorter and simpler than generalized template references.
+
+In a registry based template reference, the scheme part of the URL and the name
+of the top level template file are omitted, and the version number is delimited
+by a colon. So for example, instead of
+
+```
+https://github.com/ownerA/repository2/templateA/v1/templateA.py
```

Comment:
I don't think this is a valid URL, since it actually fetches the .html describing the file. Better example would be the one that uses the usercontent format (with branch and all the other stuff baked in too), as in the existing examples from the templates.